### PR TITLE
Update 534719-Networking ServicePoint.ConnectionLimit default behavio…

### PR DESCRIPTION
**Pull request by jerickmsft**

### CODE REVIEW

File: releases/net471/KnownIssues/534719-Networking ServicePoint.ConnectionLimit default behavior with loopback changed unexpectedly.md [hunk 1]
```diff
@@ -1,14 +1,10 @@
-# ServicePoint.ConnectionLimit default behavior with loopback changed unexpectedly
+# Changes to ServicePoint.ConnectionLimit are overriden by usage of HttpClient
 
 ## Symptoms
 The limit for HTTP connections per endpoint is controlled by the `ServicePointManager.DefaultConnectionLimit` property.
-This value defaults to 2. In the .NET Framework 4.7 and earlier versions, the limit applied only to non-loopback 
-addresses such as http://www.microsoft.com.  For loopback addresses such as http://localhost, the limit for connections
-was always int.MaxValue (2,147,483,647) unless changed by calling the `ServicePoint.ConnectionLimit` API.
+This value defaults to 2. This can be overriden programatically or in configuration for either specific hosts or for all hosts. In the .NET Framework 4.7.1 any usage of HttpClient APIs reverts any host-specific overrides to the default value of 2. This can cause applications to run slower or hang when doing multiple, parallel, requests.
```
Comment (rpetrusha) [inline releases/net471/KnownIssues/534719-Networking ServicePoint.ConnectionLimit default behavior with loopback changed unexpectedly.md]: - comma after 4.7.1
- no commas around parallel

**Comment by davidsh** [comment]
We should also change the title of the *.MD file to be more generic.

I.e. ""Networking.ServicePoint.ConnectionLimit default behavior changed.md".  This will also require the links and text to this *.MD file be changed from this page:

https://github.com/Microsoft/dotnet/blob/master/releases/net471/dotnet471-known-issues.md

**Comment by davidsh** [comment]
Otherwise, the changes LGTM

**Comment by jerickmsft** [comment]
@rpetrusha Looks good?
@davidsh  I changed the markdown link, but not sure how to test from my fork.

**Comment by rpetrusha** [comment]
Yes, LGTM, @jerickmsft. I'll merge your PR now.

**Pull request merged**